.github/workflows/run-custom-tests.yml

```diff
@@ -0,0 +1,88 @@
+
+name: Run Custom Tests
```

Comment:
Useful for debugging specific test(s) in github

Reply:
this will be useful, Thank you.